wrapper/hermes_wrapper.cpp

```diff
@@ -0,0 +1,149 @@
+  hermes_ptr->Finalize(true);
+}
+
+VBucketClass *HermesVBucketCreate(const char *name) {
+  LOG(INFO) << "Hermes Wrapper: Creating VBucket " << name << '\n';
+  hermes::api::Context ctx;
+  try {
+    hermes::api::VBucket *new_vbucket =
+      new hermes::api::VBucket(std::string(name), hermes_ptr, ctx);
+
+    return (VBucketClass *)new_vbucket;
+  }
+  catch (const std::runtime_error& e) {
+    LOG(ERROR) << "Blob runtime error\n";
+    return NULL;
+  }
+  catch (const std::length_error& e) {
+    LOG(ERROR) << "Blob length error\n";
+    return NULL;
+  }
+}
+
+void HermesVBucketLink(VBucketClass *vbkt, char *blob_name) {
+  hermes::api::VBucket *vbucket = (hermes::api::VBucket *)vbkt;
+
+  LOG(INFO) << "Hermes Wrapper: Linking Blob " << blob_name << "to VBucket "
+            << vbucket->GetName() << '\n';
+
+  hermes::api::Status status =
+    vbucket->Link(std::string(blob_name), vbucket->GetName());
+
+  if (status.Failed())
+    LOG(ERROR) << "Hermes Wrapper: HermesVBucketLink failed\n";
+}
+
+bool HermesVBucketIsValid(VBucketClass *vbkt) {
+  hermes::api::VBucket *vbucket = (hermes::api::VBucket *)vbkt;
+
+  LOG(INFO) << "Hermes Wrapper: Checking if VBucket is valid\n";
+
+  return vbucket->IsValid();
+}
+
+BucketClass *HermesBucketCreate(const char *name) {
+  LOG(INFO) << "Hermes Wrapper: Creating Bucket " << name << '\n';
+
+  try {
+    hermes::api::Bucket *new_bucket =
+      new hermes::api::Bucket(std::string(name), hermes_ptr);
+
+    return (BucketClass *)new_bucket;
+  }
+  catch (const std::runtime_error& e) {
+    LOG(ERROR) << "Blob runtime error\n";
+    return NULL;
+  }
+  catch (const std::length_error& e) {
+    LOG(ERROR) << "Blob length error\n";
+    return NULL;
+  }
+}
+
+void HermesBucketClose(BucketClass *bkt) {
+  hermes::api::Bucket *my_bkt = (hermes::api::Bucket *)bkt;
+
+  LOG(INFO) << "Hermes Wrapper: Closing Bucket " << my_bkt->GetName() << '\n';
+
+  my_bkt->Release();
+}
+
+void HermesBucketDestroy(BucketClass *bkt) {
+  hermes::api::Bucket *my_bkt = (hermes::api::Bucket *)bkt;
+
+  LOG(INFO) << "Hermes Wrapper: Destroying Bucket\n";
+
+  my_bkt->Destroy();
+}
+
+bool HermesBucketContainsBlob(BucketClass *bkt, char *name) {
+  hermes::api::Bucket *bucket = (hermes::api::Bucket *)bkt;
+
+  LOG(INFO) << "Hermes Wrapper: Checking if Bucket "
+            << bucket->GetName()
+            << " contains Blob " << name << '\n';
+
+  return bucket->ContainsBlob(name);
+}
+
+void HermesBucketPut(BucketClass *bkt, char *name, unsigned char *put_data,
+                    size_t size) {
+  hermes::api::Bucket *bucket = (hermes::api::Bucket *)bkt;
+
+  LOG(INFO) << "Hermes Wrapper: Putting Blob " << name << " to bucket " <<
+               bucket->GetName() << '\n';
+
+  hermes::api::Status status = bucket->Put(name, put_data, size);
+
+  if (status.Failed())
+    LOG(ERROR) << "Hermes Wrapper: HermesBucketPut failed\n";
+}
+
+void HermesBucketGet(BucketClass *bkt, char *blob_name, size_t kPageSize,
```

Comment:
Shouldn't `kPageSize` be `page_size`, since it's a function parameter and not a constant?